tools/FhirAnalyzer/FhirAnalyzer.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Also if this gets packaged into a nuget we'll get flagged unless it starts with `Microsoft.`